spec.bs

```diff
@@ -2867,26 +2880,24 @@ threshold when responding to [=query k-anonymity count=].
 </div>
 
 <div algorithm>
-  To <dfn>query component ad k-anonymity count</dfn> given a [=URL=] |ad|:
-    1. Let |keyString| be the [=string/concatenation=] of the following strings separated with U+000A LF:
+ To <dfn>compute the key hash of component ad</dfn> given an [=interest group ad=] |igAd|:
+   1. Let |keyString| be the [=string/concatenation=] of the following strings separated with U+000A LF:
       1. "ComponentBid"
-      1. the [=URL serializer|serialization=] of |ad|.
-    1. Let |keyHash| be the [=SHA-256=] hash of the [=ASCII encoding=] of |keyString|.
-
-    1. Return the result of [=query k-anonymity count|querying the k-anonymity count=] given |keyHash|.
+      1. the [=URL serializer|serialization=] of |igAd|.
+    1. Return the [=SHA-256=] hash of the [=ASCII encoding=] of |keyString|.
 </div>
 
 <div algorithm>
   To <dfn>query generated bid k-anonymity count</dfn> given a [=generated bid=] |bid|:
-    1. If [=query ad k-anonymity count=] given |bid|'s [=generated bid/ad descriptor=]'s
-      [=ad descriptor/url=] returns false, return false.
+    1. Compute the |adHashCode| following [=compute the key hash of ad=] with the |bid|'s [=generated bid/interest group=] and |bid|'s [=generated bid/ad descriptor=].

```

Comment:
following?

Reply:
Done

Reply:
Doesn't seem to be a change here?

Reply:
Sorry, I'm not sure what you want me to change here. This follows the same pattern as the other places "Compute x following y with w and z".

Reply:
I see. I just don't get what "following" means. If `[=compute the has key of ad=]` is an algorithm then we should just invoke the algorithm and get the result back. I was thinking something like:

```
Let |adHashCode| be the result of [=computing the key hash of ad=] given foo, bar, baz
```

... like most other algorithm invocations in the spec.

Reply:
The change on this line appears to have been reverted so this comment thread is obsolete.